library/src/webapp/editor/ckextraplugins/common-wl/js/folder-listing.js

```diff
@@ -28,9 +29,20 @@ $.fn.folderListing = function(options) {
 
   var urlPrefix = '/direct/content/resources/';
 
-  var getHtmlResults = function(json, $div) {
+  var getHtmlResults = function(json, $div, root) {
     var html = $('<ul/>').addClass('jqueryFileTree').hide();
-
+    var urlRoot = json[0].resourceId.trim();
+    if(settings.displayRootDirectory && urlRoot === root ){
```

Comment:
when we are showing root folder we want both 'html' and 'ul', as this 'ul' is new list for the root, I am not sure if we can use just one variable